cmd_gen_password.go

```diff
@@ -73,7 +73,7 @@ func (x *genPasswordCommand) Execute(_ []string) error {
 		}
 	}
 
-	fmt.Printf("%s\n", passwordString)
```

Comment:
This is just a defense in depth fix, and the seeds actually weren't affected by this issue right? Or I guess maybe eventually a deployed seed would, as it's a random event that the characters line up

Reply:
Just to be a bit more consistent. A tool that is sensitive to newline characters in certain things should also be careful about not writing them by default. Also, we have comments like "lndinit doesn't add newlines itself" in some places. So I think this is just best practice.